frontend/src/modules/search/components/SearchBox.js

```diff
@@ -80,12 +81,14 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
     updateFiltersFromURLParams = () => {
         const props = this.props;
 
-        const statuses = this.getInitialStatuses();
-        if (props.parameters.status) {
-            props.parameters.status.split(',').forEach(f => {
-                statuses[f] = true;
-            });
-        }
+        // let statuses = this.getInitialStatuses();
```

Comment:
@adngdb can I remove this code? Because the status state is updated by `getDerivedStateFromProps`.
Or I just uncomment and leave it as it was?

Reply:
Unless there's a good reason, it's better not to leave commented code around.